api/controllers/UserController.js

```diff
@@ -87,7 +85,15 @@ module.exports = {
         });
       }
 
-      const game = populatedGame ?? unpopulatedGame;
+      if (unpopulatedGame && unpopulatedGame.lastEvent.victory) {
```

Comment:
We can optionally chain this for brevity
```suggestion
      if (unpopulatedGame?.lastEvent?.victory) {
```

Reply:
Also let's put this right below the other unpopulatedGame conditional just to group the similar ones